projects/ccf-rui/src/app/shared/components/gender-selector/gender-selector.component.theme.scss

```diff
@@ -0,0 +1,29 @@
+@use '~@angular/material/theming' as *;
+
+@mixin theme($theme) {
+  $foreground: map-get($theme, foreground);
+  $background: map-get($theme, background);
+  $primary: map-get($theme, primary);
+
+  .gender-slider {
```

Comment:
These rules become global non-encapsulated therefore they need to be wrapped inside a css selector for this specific component. Check right-sidebar code + theme on how it is done.